testsuite/resources/HelloWorld.mos

```diff
@@ -0,0 +1,13 @@
+loadString("
+model HelloWorld
+  parameter Real a = -1;
+  parameter Real x_start = 1;
+  Real x(start=x_start, fixed=true);
+equation
+  der(x) = a*x;
+end HelloWorld;
+"); getErrorString();
+
+buildModelFMU(HelloWorld, version="2.0", fmuType="me_cs", fileNamePrefix="HelloWorld", platforms={"x86_64-linux-gnu"}); getErrorString();
+
+system("unzip HelloWorld.fmu -d HelloWorld/");
```

Comment:
Please remove the sources inside the fmu.

```bash
system("zip -qd HelloWorld.fmu sources/* && unzip HelloWorld.fmu -d HelloWorld/");
```

Reply:
Hm, I first used this line (copied it form other files), but got an error when executing it on my machine. I will try once more

Reply:
Worked no without an error, whatever I did wrong the first time. I'll update the PR.